src/components/Filters/FiltersList.vue

```diff
@@ -95,10 +100,23 @@ export default {
 </script>
 
 <style lang="scss" scoped>
+.filterlist-header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-left: 52px;
+  padding-top: 31px;
+  padding-bottom: 16px;
```

Comment:
When we see a lot of pixel values like this, we tend to refer to them as "magic numbers". These are numbers that might look correct but don't correspond to any _system_ of spacing in the design, and are often a sign of problems. We're also trying to stick to REM units instead of pixel values. On this site 1rem = 16px, so you could do a little math to convert these.

Reply:
I can see why having literal values in styles might make for a less scalable and functional website! I'll swap those all over :)